### PR TITLE
Import figure tests from Scott O'hara 

### PR DESCRIPTION
**Pull request by mfairchild365**
This imports tests from Scott O'Hara: https://scottaohara.github.io/testing/figure/

Note: sample script used to import the tests: https://gist.github.com/mfairchild365/7cf5425b1950182e25198605fe7c9f00

In order for this to work, the script must be in the `/scripts` directory and the original test page must be saved to `/tests/html/html_figure_results.html`

### CODE REVIEW

**Comment by mfairchild365** [comment]
@scottaohara could you give this a once over?

File: data/tests/tech/html/html_figure/test_0.json [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+        }
+      }
+    },
+    "narrator": {
+      "browsers": {
+        "edge": {
+          "at_version": "1804",
+          "browser_version": "42",
+          "os_version": "1804",
+          "date": "2019-01-14",
```
Comment (scottaohara) [inline data/tests/tech/html/html_figure/test_0.json]: 2019-01-21

File: data/tests/tech/html/html_figure/test_0.json [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+        }
+      }
+    },
+    "nvda": {
+      "browsers": {
+        "firefox": {
+          "at_version": "2018.4.1",
+          "browser_version": "64.0.2",
+          "os_version": "1809",
+          "date": "2019-01-14",
```
Comment (scottaohara) [inline data/tests/tech/html/html_figure/test_0.json]: 2019-01-21

File: data/tests/tech/html/html_figure/test_0.json [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+        }
+      }
+    },
+    "vo_macos": {
+      "browsers": {
+        "safari": {
+          "at_version": "10.14",
+          "browser_version": "12.0.2",
+          "os_version": "10.14",
+          "date": "2019-01-14",
```
Comment (scottaohara) [inline data/tests/tech/html/html_figure/test_0.json]: 2019-01-21

File: data/tests/tech/html/html_figure/test_0.json [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+          ]
+        }
+      }
+    },
+    "vo_macos": {
+      "browsers": {
+        "safari": {
+          "at_version": "10.14",
+          "browser_version": "12.0.2",
+          "os_version": "10.14",
```
Comment (scottaohara) [inline data/tests/tech/html/html_figure/test_0.json]: 10.14.2 to be specific

File: data/tests/tech/html/html_figure/test_0.json [hunk 5]
```diff
@@ -0,0 +1,141 @@
...
+        }
+      }
+    },
+    "vo_ios": {
+      "browsers": {
+        "ios_saf": {
+          "at_version": "12.1",
+          "browser_version": "12.1.2",
+          "os_version": "12.1",
+          "date": "2019-01-14",
```
Comment (scottaohara) [inline data/tests/tech/html/html_figure/test_0.json]: 2019-01-21

File: data/tests/tech/html/html_figure/test_0.json [hunk 6]
```diff
@@ -0,0 +1,141 @@
...
+        }
+      }
+    },
+    "talkback": {
+      "browsers": {
+        "and_chr": {
+          "at_version": "7.2",
+          "browser_version": "70",
+          "os_version": "7.2",
+          "date": "2019-01-14",
```
Comment (scottaohara) [inline data/tests/tech/html/html_figure/test_0.json]: 2019-01-21

File: data/tests/tech/html/html_figure/test_0.json [hunk 7]
```diff
@@ -0,0 +1,141 @@
...
+          ]
+        }
+      }
+    },
+    "talkback": {
+      "browsers": {
+        "and_chr": {
+          "at_version": "7.2",
+          "browser_version": "70",
+          "os_version": "7.2",
```
Comment (scottaohara) [inline data/tests/tech/html/html_figure/test_0.json]: 8.1

File: data/tests/tech/html/html_figure/test_1.json [hunk 8]
```diff
@@ -0,0 +1,141 @@
...
+      "date": "2019-01-21",
+      "message": "Test created. Thank you @scottaohara."
+    }
+  ],
+  "at": {
+    "jaws": {
+      "browsers": {
+        "ie": {
+          "at_version": "2019",
+          "browser_version": "44",
```
Comment (scottaohara) [inline data/tests/tech/html/html_figure/test_1.json]: 11??

File: data/tests/tech/html/html_figure/test_1.json [hunk 9]
```diff
@@ -0,0 +1,141 @@
...
+          "os_version": "1809",
+          "date": "2019-01-14",
```
Comment (scottaohara) [inline data/tests/tech/html/html_figure/test_1.json]: 2019-01-21

File: data/tests/tech/html/html_figure/test_1.json [hunk 10]
```diff
@@ -0,0 +1,141 @@
...
+              "output": "(unknown)",
+              "result": "fail"
+            }
+          ]
+        },
+        "firefox": {
+          "at_version": "2019",
+          "browser_version": "64.0.2",
+          "os_version": "1809",
+          "date": "2019-01-14",
```
Comment (scottaohara) [inline data/tests/tech/html/html_figure/test_1.json]: 2019-01-21

File: data/tests/tech/html/html_figure/test_1.json [hunk 11]
```diff
@@ -0,0 +1,141 @@
...
+        }
+      }
+    },
+    "narrator": {
+      "browsers": {
+        "edge": {
+          "at_version": "1804",
+          "browser_version": "42",
+          "os_version": "1804",
+          "date": "2019-01-14",
```
Comment (scottaohara) [inline data/tests/tech/html/html_figure/test_1.json]: 2019-01-21

File: data/tests/tech/html/html_figure/test_1.json [hunk 12]
```diff
@@ -0,0 +1,141 @@
...
+        }
+      }
+    },
+    "nvda": {
+      "browsers": {
+        "firefox": {
+          "at_version": "2018.4.1",
+          "browser_version": "64.0.2",
+          "os_version": "1809",
+          "date": "2019-01-14",
```
Comment (scottaohara) [inline data/tests/tech/html/html_figure/test_1.json]: 2019-01-21

File: data/tests/tech/html/html_figure/test_1.json [hunk 13]
```diff
@@ -0,0 +1,141 @@
...
+          ]
+        }
+      }
+    },
+    "vo_macos": {
+      "browsers": {
+        "safari": {
+          "at_version": "10.14",
+          "browser_version": "12.0.2",
+          "os_version": "10.14",
```
Comment (scottaohara) [inline data/tests/tech/html/html_figure/test_1.json]: 10.14.2

File: data/tests/tech/html/html_figure/test_1.json [hunk 14]
```diff
@@ -0,0 +1,141 @@
...
+          "date": "2019-01-14",
```
Comment (scottaohara) [inline data/tests/tech/html/html_figure/test_1.json]: 2019-01-21

File: data/tests/tech/html/html_figure/test_1.json [hunk 15]
```diff
@@ -0,0 +1,141 @@
...
+        }
+      }
+    },
+    "vo_ios": {
+      "browsers": {
+        "ios_saf": {
+          "at_version": "12.1",
+          "browser_version": "12.1.2",
+          "os_version": "12.1",
+          "date": "2019-01-14",
```
Comment (scottaohara) [inline data/tests/tech/html/html_figure/test_1.json]: 2019-01-21

File: data/tests/tech/html/html_figure/test_1.json [hunk 16]
```diff
@@ -0,0 +1,141 @@
...
+          ]
+        }
+      }
+    },
+    "talkback": {
+      "browsers": {
+        "and_chr": {
+          "at_version": "7.2",
+          "browser_version": "70",
+          "os_version": "7.2",
```
Comment (scottaohara) [inline data/tests/tech/html/html_figure/test_1.json]: 8.1

File: data/tests/tech/html/html_figure/test_10.json [hunk 17]
```diff
@@ -0,0 +1,141 @@
...
+          ]
+        }
+      }
+    },
+    "talkback": {
+      "browsers": {
+        "and_chr": {
+          "at_version": "7.2",
+          "browser_version": "70",
+          "os_version": "7.2",
```
Comment (scottaohara) [inline data/tests/tech/html/html_figure/test_10.json]: 8.1

File: data/tests/tech/html/html_figure/test_11.json [hunk 18]
```diff
@@ -0,0 +1,140 @@
...
+          ]
+        }
+      }
+    },
+    "vo_macos": {
+      "browsers": {
+        "safari": {
+          "at_version": "10.14",
+          "browser_version": "12.0.2",
+          "os_version": "10.14",
```
Comment (scottaohara) [inline data/tests/tech/html/html_figure/test_11.json]: 10.14.2

File: data/tests/tech/html/html_figure/test_11.json [hunk 19]
```diff
@@ -0,0 +1,140 @@
...
+          ]
+        }
+      }
+    },
+    "talkback": {
+      "browsers": {
+        "and_chr": {
+          "at_version": "7.2",
+          "browser_version": "70",
+          "os_version": "7.2",
```
Comment (scottaohara) [inline data/tests/tech/html/html_figure/test_11.json]: 8.1

**Pull request merged**